### PR TITLE
Add Engineering Manager JD

### PR DESCRIPTION
**Pull request by elle**

### CODE REVIEW

File: roles/engineering_manager.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- Merged PRs auto-deploy to staging and production via chatops.
+- Recorded work in progress demos are shared with the company.
+
+**Key responsibilities:**
+
+- Lead the main engineering team to develop scalable, reliable, and maintainable systems
+- Promote good engineering practices within the team and beyond
+- Identify and implement improvements to our products and processes to support our growth
+- Collaborate with cross-functional teams to ensure technical solutions align with product strategy
+- Ensure team members understand how the work they are doing ties to wider company goals
```
Comment (elle) [inline roles/engineering_manager.md:43]: Or maybe "Roadmap and backlog management; communicating priorities to team and keeping on track"

**Review by elle — COMMENTED** [review]
What else am I missing?

File: roles/engineering_manager.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+- Ideas and priorities flow from across the company.
+- Big initiatives are planned into 4-8 week releases.
+- Progress is tracked in two-week cycles.
+- Merged PRs auto-deploy to staging and production via chatops.
+- Recorded work in progress demos are shared with the company.
+
+**Key responsibilities:**
+
+- Lead the main engineering team to develop scalable, reliable, and maintainable systems
```
Comment (thephw) [inline roles/engineering_manager.md]:
```suggestion
- Empower the engineering team to develop scalable, reliable, and maintainable systems
```

File: roles/engineering_manager.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- Promote good engineering practices within the team and beyond
+- Identify and implement improvements to our products and processes to support our growth
+- Collaborate with cross-functional teams to ensure technical solutions align with product strategy
```
Comment (thephw) [inline roles/engineering_manager.md]:
```suggestion
- Collaborate with cross-functional teams to ensure technical solutions align with product strategy and reasonable timelines
```

File: roles/engineering_manager.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+- Ideas and priorities flow from across the company.
+- Big initiatives are planned into 4-8 week releases.
+- Progress is tracked in two-week cycles.
+- Merged PRs auto-deploy to staging and production via chatops.
+- Recorded work in progress demos are shared with the company.
+
+**Key responsibilities:**
+
+- Lead the main engineering team to develop scalable, reliable, and maintainable systems
+- Promote good engineering practices within the team and beyond
```
Comment (thephw) [inline roles/engineering_manager.md:39]:
```suggestion
- Contribute via leveraged output activities -- one-on-ones, training, pair programming, and adding CI checks.
- Promote good engineering practices within the team and beyond
```

File: roles/engineering_manager.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+- Identify and implement improvements to our products and processes to support our growth
+- Collaborate with cross-functional teams to ensure technical solutions align with product strategy
+- Ensure team members understand how the work they are doing ties to wider company goals
+- Manage a remote team across multiple timezones
+- Nurture a healthy team culture by
+  - role modelling the behaviours you wish to see,
```
Comment (thephw) [inline roles/engineering_manager.md]:
```suggestion
  - being a role model for the behaviors you wish to see,
```

File: roles/engineering_manager.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  - coach team members around work and personal growth,
+  - seek to create and support a diverse and inclusive team, and
+  - monitor team health and morale and ensure improvements
+- Foster a continuous learning and improvement mindset within your team. Ensure team members have the skills needed for their role
+- Be a sponsor for engineers so they have access to growth opportunities they wouldn’t otherwise have on their own
+
+**Your peers would describe you as:**
+
+- Dependable in meeting your commitments
```
Comment (thephw) [inline roles/engineering_manager.md:55]:
```suggestion
- Extremely dependable in meeting your commitments
```

Comment (elle) [reply roles/engineering_manager.md:55]: Only one I'm going to keep as is. I don't think the Extremely is needed

**Pull request merged**